### DISCUSSION TITLE
Create developer environments on behalf of user with PAC CLI

### DISCUSSION THREAD

**Discussion by anttipajunen** [microsoft/powerplatform-build-tools#563] [Ideas]
Creating dev envs on behalf of users just landed in PPAC. What are the plans to enable creation of dev envs on behalf of users by using the PAC CLI?

**Comment by snizar007** [comment]
Do you have the link to this PPAC feature handy?

**Comment by petrochuk** [comment]
We are planning to add this functionality in the next release of PAC CLI.

Thanks,
Andrew
________________________________
From: Antti Pajunen ***@***.***>
Sent: Friday, September 29, 2023 3:42 AM
To: microsoft/powerplatform-build-tools ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [microsoft/powerplatform-build-tools] Create developer environments on behalf of a user with PAC CLI (Discussion #563)


Creating dev envs on behalf of users just landed in PPAC. What are the plans to enable creation of dev envs on behalf of users by using the PAC CLI?

**Comment by marcelbf** [comment]
done in 1.28
 - Create command has new --user parameter to specify the owner of the developer environment
